concordion/src/test/resources/nl/knaw/huc/textrepo/rest/TestRestDocumentFiles.md

```diff
@@ -46,3 +46,21 @@ Then:
  - Full response:
 
 [ ](- "ext:embed=#paginateResult.body")
+
+### Filter by file type
+Results can be filtered by type.
+
+When retrieving the versions of a file with a `GET` to [`/rest/documents/{id}/files?typeId={typeId}`](- "#getEndpoint") 
+
+ - where `{typeId}` is [ ](- "ext:embed=code(getTextTypeId())");
```

Comment:
Hier nog even
`- where `{id}` is [ ](- "ext:embed=code(getDocId())");`
toevoegen zoals erboven bij "Paginate document files"

Reply:
Gefixt in 4888f2e